.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "lldb",
```

Comment:
What's the purpose of this file?

Reply:
It's the debugger config for vscode https://code.visualstudio.com/docs/editor/debugging
It tells vscode how to spawn dust-api for debugging 

IMO It makes sense to share tooling configs even if we're not using the same editor.

Otherwise, I can add a line to .gitignore.